rhc-ose-ansible/roles/docker/templates/docker-storage-setup.j2

```diff
@@ -0,0 +1,2 @@
+DEVS={{ docker_storage_block_device }}
```

Comment:
This may be a bit of a lower priority, but I think it would be good to support option B as well - i.e: using an existing volume group. In that scenario, the DEVS entry should be omitted. 
```
https://docs.openshift.com/enterprise/3.1/install_config/install/prerequisites.html#configuring-docker-storage
```